app/models/archiver.rb

```diff
@@ -3,11 +3,15 @@
 module Archiver
   ARCHIVE_TYPES = [
     { source: Institution, archive: InstitutionsArchive },
-    { source: InstitutionProgram, archive: InstitutionProgramsArchive },
     { source: ZipcodeRate, archive: ZipcodeRatesArchive },
+    { source: InstitutionProgram, archive: InstitutionProgramsArchive },
     { source: VersionedSchoolCertifyingOfficial, archive: VersionedSchoolCertifyingOfficialsArchive }
   ].freeze
 
+  ARCHIVE_TYPES_BY_VERSION_ID = [].freeze
+
+  ARCHIVE_TYPES_BY_PARENT_ID = [].freeze
```

Comment:
Both of these arrays are empty and frozen, what is the purpose of them?

Reply:
I was on the fence about leaving the content of these present and commented out or just empty.

This story is dependent on changes to be implemented by:
 - https://github.com/department-of-veterans-affairs/va.gov-team/issues/4044 
 - https://github.com/department-of-veterans-affairs/va.gov-team/issues/4063. 

Once those changes are present these arrays will look more like:

```
  ARCHIVE_TYPES_BY_VERSION_ID = [
   { source: Institution, archive: InstitutionsArchive, versions: Version},
   { source: ZipcodeRate, archive: ZipcodeRatesArchive, versions: Version}
  ].freeze

  ARCHIVE_TYPES_BY_PARENT_ID = [
   { source: InstitutionProgram, archive: InstitutionProgramsArchive, versions: Version, institutions: Institution},
   { source: VersionedSchoolCertifyingOfficial, archive: VersionedSchoolCertifyingOfficialsArchive, versions: Version, institutions: Institution}
  ].freeze
```